pep-0012.rst

```diff
@@ -113,20 +113,25 @@ directions below.
   first appearance in.  Do not use an alpha or beta release
   designation here.  Thus, if the last version of Python was 2.2 alpha
   1 and you're hoping to get your new feature into Python 2.2, set the
-  header to::
+  header to:
+
+  .. code-block:: email
 
       Python-Version: 2.2
 
-- Leave Post-History alone for now; you'll add dates to this header
-  each time you post your PEP to the designated discussion forum (and announce
-  it on Python-Dev, if needed; see the ``Discussions-To`` header above).
+- Leave Post-History alone for now; you'll add dates and corresponding links
+  to this header each time you post your PEP to the designated discussion forum
+  (and update the ``Discussions-To`` header with said link, as above).
   If you posted threads for your PEP on August 14, 2001 and September 3, 2001,
-  the Post-History header would look like::
+  the Post-History header would look something like:
+
+  .. code-block:: email
 
-      Post-History: 14-Aug-2001, 03-Sept-2001
+      Post-History: `14-Aug-2001 <https://www.example.com/thread_1>`_,
+                    `03-Sept-2001 <https://www.example.com/thread_2>`_
 
-  You must manually add new dates and commit them (with a pull request
-  if you don't have push privileges).
+  You must manually add new dates/links and commit them as soon as you post a
+  new discussion thread (with a pull request, if you don't have push rights).
```

Comment:
```suggestion
  You must manually add new dates/links as soon as you post a
  new discussion thread.
```
This is just redundant wording. We don't have to reiterate how to make changes to a PEP.